os/include/tinyara/pm/pm.h

```diff
@@ -197,6 +197,17 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
+/* This enumeration provides all power management related wakeup source code. */
+
+typedef enum {
+	PM_WAKEUP_BLE,           /* BLE Interrupts */
+	PM_WAKEUP_WIFI,          /* Wifi Interrupts */
+	PM_WAKEUP_UART,          /* UART Interrupts */
```

Comment:
Does UART share the same wakeup reason as LOGUART?

Reply:
How about rename UART_CONSOLE, UART_TTYS2